spec/unit/x509/cert_provider_spec.rb

```diff
@@ -524,4 +524,30 @@ def expects_private_file(path)
       end
     end
   end
+
+  context 'CRL last update time' do
+    let(:crl_path) { tmpfile('pem_crls') }
+
+    it 'returns nil if the CRL does not exist' do
+      provider = create_provider(crlpath: '/does/not/exist')
+
+      expect(provider.crl_last_update).to be_nil
+    end
+
+    it 'returns the last update time' do
+      now = Time.now
+      Puppet::FileSystem.touch(crl_path, mtime: now)
```

Comment:
As things are now, this probably doesn't need to be setting a time in the past like the "sets the last update time" test does, but I'd probably be more comfortable if it did so that we could differentiate between the mtime that was set from this touch vs. anything that happened as a side effect of creating a `CertProvider`.